docs/sources/shared/auth/intro.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Intro paragraph
+---
+
+Grafana supports many ways to authenticate users. The authentication setup determines which users have access to Grafana and how they log in to Grafana.
+You can also configure Grafana to manage users' roles and team memberships based on the information returned by the auth provider integration.
```

Comment:
I'm not sure what this means. Is this about using Grafana as the auth provider? Or is this something else?